docs/registry-docs/copying-images.md

```diff
@@ -1,198 +1,143 @@
 
-## Rancher
+# fetch the content from the carbide secured registry
+# verify the version, location of the key, and the platform/architecture
+hauler store sync --products longhorn=v1.6.0 --key carbide-key.pub --platform linux/amd64
 
-### Cert Manager
+# copy the content from the hauler store to your registry
+hauler store copy --username <redacted> --password <redacted> registry://<registry-url>
+```
+
+## NeuVector
 
 ```bash
-# Log into the registry
+# authenticate into carbide secured registry
 cosign login -u <redacted> -p <redacted> rgcrprod.azurecr.us
 
-# Your target registry (and login if it requires authentication)
-TARGET_REGISTRY=YOUR_REGISTRY_DOMAIN_HERE
-# cosign login -u YOUR_USER -p YOUR_PASSWORD $TARGET_REGISTRY
-
-# Rancher supports v1.7.1 currently. See here for more info:
-# https://ranchermanager.docs.rancher.com/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster#4-install-cert-manager
-CERT_MANAGER_RELEASE="v1.7.1"
+# download the public key for carbide
+curl -sfOL https://raw.githubusercontent.com/rancherfederal/carbide-releases/main/carbide-key.pub
 
-# Add the cert manager repo (required Helm)
-helm repo add jetstack https://charts.jetstack.io
-helm repo update
+# fetch the content from the carbide secured registry
+# verify the version, location of the key, and the platform/architecture
+hauler store sync --products neuvector=v5.3.0 --key carbide-key.pub --platform linux/amd64
 
-# Grab the list of images and download them (requires docker, grep, sed, and awk)
-for image in $(helm template jetstack/cert-manager --version $CERT_MANAGER_RELEASE | grep 'image:' | sed 's/"//g' | awk '{ print $2 }'); do
-    source_image=$(echo $image | sed 's/quay.io/rgcrprod.azurecr.us/g')
-    dest_image=$(echo $image | sed "s/quay.io/$TARGET_REGISTRY/g")
-    cosign copy $source_image $dest_image
-done
+# copy the content from the hauler store to your registry
+hauler store copy --username <redacted> --password <redacted> registry://<registry-url>
 ```
 
-See [Rancher Manager Configuration](rancher-config.md) for configuring the Cert Manager chart to use these images.
-
-### Rancher
+## Kubewarden
 
 ```bash
-# Log into the registry
+# authenticate into carbide secured registry
 cosign login -u <redacted> -p <redacted> rgcrprod.azurecr.us
 
-# Your target registry (and login if it requires authentication)
-TARGET_REGISTRY=YOUR_REGISTRY_DOMAIN_HERE
-# cosign login -u YOUR_USER -p YOUR_PASSWORD $TARGET_REGISTRY
+# download the public key for carbide
+curl -sfOL https://raw.githubusercontent.com/rancherfederal/carbide-releases/main/carbide-key.pub
 
-# Set the specific release of Rancher you're targeting: https://github.com/rancher/rancher/releases
-RANCHER_RELEASE=v2.7.1
+# fetch the content from the carbide secured registry
+# verify the version, location of the key, and the platform/architecture
+hauler store sync --products kubewarden=v2.0.8 --key carbide-key.pub --platform linux/amd64
```

Comment:
Should be...
`hauler store sync --products kubewarden=kubewarden-controller-2.0.8 --key carbide-key.pub --platform linux/amd64`

I think you fixed this in the air gapped section but missed it here.

Reply:
thanks for catching it! commit incoming...